css/main.css

```diff
@@ -654,6 +790,88 @@ header a {
     bottom: 175px;
     width: 130.28px;
   }
+
+  .project-pop-up-window {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    background: #c1c7d0;
+    mix-blend-mode: multiply;
+    height: 100vh;
+    width: 100vw;
+    position: fixed;
+    top: 0;
+  }
+
+  .project-pop-up-window > div {
+    width: 60%;
+    height: 90%;
+    padding: 16px;
+    background-color: white;
+    border: 1px solid #dfe1e6;
+    border-radius: 16px;
+    display: grid;
+    grid-template-columns: repeat(5, auto);
+    grid-template-areas: 'close close close close close' 'photo photo photo photo photo' 'h h h btn btn' 'skill skill skill skill skill' 'txt txt txt txt txt';
+    overflow: auto;
+  }
+
+  .project-pop-up-window > div > img {
+    display: block;
+    margin: 0;
+    padding: 0;
+    grid-area: close;
+    justify-self: flex-end;
+  }
+
+  .project-pop-up-window .close-item {
+    background-repeat: no-repeat;
+    background-size: 100% 100%;
+    min-height: 15rem;
+    position: relative;
+    grid-area: photo;
+  }
```

Comment:
Great design @mohamed20163858 🤝

- [ ] You may kindly fix your popup images as they appear to be stretched. You may increase their height and make them take up most of the space on the popup windows just as they appear in the Figma design.
- Your design
![desktop image](https://user-images.githubusercontent.com/63917725/180092025-a3309637-c7f6-4360-9390-528243f343e0.PNG)

- Figma design
![Figma desktop image](https://user-images.githubusercontent.com/63917725/180092446-1a336805-425d-4925-8dac-4e1dc4f511c6.PNG)

Reply:
@1GORDON thanks  a lot for mentioning that i already resolved this problem